packages/app/src/lib/components/passPromt.svelte

```diff
@@ -13,17 +13,21 @@
 	import { onMount } from 'svelte'
 	import { toast } from 'svelte-sonner'
 
+	import Spinner from './assets/spinner.svelte'
 	import Pattern from './Pattern.svelte'
 	import Checkbox from './ui/checkbox/checkbox.svelte'
 	import CAvatar from './ui/custom-components/c-avatar.svelte'
 	import Label from './ui/label/label.svelte'
 
 	export let dialogOpen = false
 	export let accointInfo: NsecAccount
+	let loading = false
```

Comment:
i think the inconsistency between naming of "isLoading" and "loading" does not look that good!

Reply:
nice